.github/workflows/dev-deploy-cloud-functions.yml

```
@@ -0,0 +1,89 @@
+          --runtime=python310 \
+          --source=cloud_functions/${{ matrix.function }} \
+          --trigger-http \
+          --entry-point=${{ matrix.function }} \
+          --no-allow-unauthenticated \
+          --max-instances=10 \
+          --min-instances=0 \
+          --memory=256MB \
+          --set-env-vars=GOOGLE_CLOUD_PROJECT=${{ env.PROJECT_ID }},ENVIRONMENT=${{ env.ENVIRONMENT }} \
+          --service-account=${{ secrets.GCP_DP_SA_EMAIL }} \
+          --set-custom-audiences=${{ env.PROJECT_ID }}
+        
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)